get-pull-request/action.yml

```diff
@@ -7,7 +7,7 @@ inputs:
     required: true
     default: ""
   filter-out-closed:
-    description: Whether to ignore closed pull requests
+    description: "Whether to ignore closed pull requests. If the commit is not present in the default branch, closed pull requests cannot be found."
```

Comment:
Aha, hadde dette konsekvenser for noe?

Reply:
Ja, det betyr at `8BitJonny/gh-get-current-pr` ikke finner PR-en hvis den er lukket uten å ha blitt merget, ref. https://github.com/biblioteksentralen/libry-content/pull/424